views/index.jade

```diff
@@ -40,6 +39,7 @@ block content
             option(value='pyramid') pyramid
             option(value='bizstyle') bizstyle
             option(disabled='') PyPi Custom Themes
+            option(value='sphinx_fossasia_theme') sphinx_fossasia_theme
```

Comment:
how about we put this at top under heading `FOSSASIA's custom theme`? Will it make sense?